e2e/tests/Dashboard/AllElementsPresentNoTrigger.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { test } from '@playwright/test';
+import AggregateComponenet from 'Pages/AggregateComponenet';
+import ChatComponent from 'Pages/ChatComponent';
+import DashboardPage from 'Pages/DashboardPage';
+import HeaderComponent from 'Pages/HeaderComponent';
+import MapComponenet from 'Pages/MapComponenet';
+import TopBarComponent from 'Pages/TopBarComponent';
+import { NoTriggerDataSet } from 'testData/testData.enum';
+
+import { FloodsScenario } from '../../../services/API-service/src/scripts/enum/mock-scenario.enum';
+import {
+  getAccessToken,
+  mockFloods,
+  resetDB,
+} from '../../../services/API-service/test/helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Login
+  const loginPage = new LoginPage(page);
+
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+  // We should maybe create one mock for all different disaster types for now we can just use floods
+  await mockFloods(
+    FloodsScenario.NoTrigger,
+    NoTriggerDataSet.CountryCode,
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(
+    NoTriggerDataSet.UserMail,
+    NoTriggerDataSet.UserPassword,
+  );
+});
+
+test('[30509] All Dashboard elements are present in no-trigger mode', async ({
```

Comment:
What is the number 30509? Is it linked to the Test Plan on Azure?

Reply:
It is linked to Azure test plan for now. The same test description I add now in the Github issue item.